src/Service.GraphQLBuilder/Mutations/CreateMutationBuilder.cs

```diff
@@ -249,7 +254,7 @@ private static NameNode GenerateInputTypeName(string typeName)
                 entities);
 
             // Create authorize directive denoting allowed roles
```

Comment:
nit: this comment applies to CreateAuthorizationDirectiveIfNecessary() call below